docs/variants.md

```diff
@@ -65,11 +65,20 @@ This stream strategy should be used together with event-sourcing, if you use one
 different aggregates named `user-123`, `user-234`, `todo-345` and `todo-456`, you would have 4 different event streams,
 one for each aggregate.
 
-This stream strategy is the most performant of all, but it will create a lot of database tables, which is something not
+This stream strategy is the most performant of all (with downsides, see notes), but it will create a lot of database tables, which is something not
 everyone likes (especially DB admins).
 
 All needed database tables will be created automatically for you.
 
+Note: For event-store projections the aggregate stream strategy is not that performant anymore, consider using [CategoryStreamProjectionRunner](https://github.com/prooph/standard-projections/blob/master/src/CategoryStreamProjectionRunner.php) from the [standard-projections]((https://github.com/prooph/standard-projections) repository.
+But even than, the projections would be slow, because the projector would need to check all streams for new events, let's take this example:
+- Each check is an SQL query, well let's say each check takes 1ms.
```

Comment:
You increased this estimate 10 times so the numbers below don't match anymore.

I'm not sure we should put such numbers to documentation either. As you said, we don't have a benchmark, it's just very rough estimate (in reality the times would probably be higher).

How about this?

```
But even than, the projections would be slow, because the projector needs to check all the streams one-by-one for any new events. Because of this speed of finding and projecting any new events depends on the number of streams which means it would rapidly decrease as you add more data to your event store.

You could however drastically improve the projections, if you would add a category stream projection as event-store-plugin. (This doesn't exist, yet)
```

Reply:
accepted :)